packages/utils/src/saml/bulk-saml.ts

```diff
@@ -0,0 +1,47 @@
+// This is a helper script that lets you send xcpd requests in bulk
+import fs from "fs";
+
+import { createAndSignBulkXCPDRequests } from "@metriport/core/external/saml/xcpd/iti55-envelope";
+import { sendSignedRequests } from "@metriport/core/external/carequality/ihe-gateway-v2/saml-client";
+import { getEnvVarOrFail } from "@metriport/core/util/env-var";
+
+import * as dotenv from "dotenv";
+dotenv.config();
```

Comment:
Let's be careful w/ imports, please:
- Dotenv should be the first import (we usually even have a comment to avoid it being moved around);
- Imports should not have empty lines, so we can auto organize them with VSCode;

Let's have clearer comments for the script after the imports and before the const/code - there are multiple examples of this (e.g., `get-consolidated.ts`).